autoconfigure/storage-logzio/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.logz.zipkin</groupId>
+        <artifactId>zipkin-logzio-autoconfigure</artifactId>
+        <!--suppress MavenPropertyInParent -->
```

Comment:
note: this approach I carried over, but it is discouraged and results in warnings. It is probably better to use normal mechanisms.. eg hard-set the version and let the release plugin update it consistently.